src/Controllers/AuthController.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace Railroad\MusoraApi\Controllers;
+
+use Exception;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Illuminate\Routing\Controller;
+use Railroad\Ecommerce\Events\AppSignupStartedEvent;
+use Railroad\MusoraApi\Contracts\UserProviderInterface;
+use Railroad\MusoraApi\Requests\CreateIntercomUserRequest;
+
+class AuthController extends Controller
```

Comment:
Controller class name may be updated to reflect all controller actions. AvatarController action may be refactored in here, to keep all user account related actions in one place.